app/models/content_page.rb

```diff
@@ -35,4 +35,8 @@ def module_item
   def formative?
```

Comment:
Let's delete this method as we only want to ask if a module is summative or not.

Reply:
Also `content_page_spec` 
```
  # describe '#formative?' do
  # end
```

Reply:
My understanding from Laura's document is that we do want to ask if something has formative... but I will leave it for you to ask and determine the accuracy of that statment.